install/advanced/core/index.rst

```diff
@@ -1370,34 +1884,34 @@ Now the geonode will run on your windows.
 Postgresql Database setup
 ^^^^^^^^^^^^^^^^^^^^^^^^^
 
-In this section we are going to install setup ``PostgreSQL`` database on GeoNode. GeoNode uses the ``PostgreSQL 11`` database. 
+In this section we are going to install setup ``PostgreSQL`` database on GeoNode. GeoNode uses the ``PostgreSQL 11`` database.
```

Comment:
Version of PostgreSQL is outdated here

Reply:
> Version of PostgreSQL is outdated here

Nope, that's the windows section, enterprise db link points to 11